base/config_files/docker_run.sh

```diff
@@ -7,7 +7,11 @@ if [ "$DB_SERVER" = "<to be defined>" -a $PS_INSTALL_AUTO = 1 ]; then
 	exit 1
 fi
 
-if [ ! -f ./config/settings.inc.php  ]; then
+if [ ! -f ./config/settings.inc.php ] && [ ! -f ./install.lock ]; then
+
+        echo "\n* Setting up install lock file..."
```

Comment:
Could you fix the indentation?
One commit to replace spaces with tabs, or better, use only spaces in the whole file :D

Reply:
Just did that, let me know if there's any remaining indentation issue.

Reply:
:+1: